### PR TITLE
no DD_ prefix in tag name

### PR DESCRIPTION
**Pull request by lpriima**

### CODE REVIEW

File: src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java [hunk 1]
```diff
@@ -62,10 +62,10 @@
...
 
     enum Literal {
-        service,
-        env,
-        version,
-        tags;
+        SERVICE,
+        ENV,
+        VERSION,
+        TAGS;
```
Comment (jdgumz) [inline src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java:68]: What is the function of `TAGS`? Do we end up with a tag that has this sort of structure: `tags:<DD_TAGS>`?

Comment (lpriima) [reply src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java:68]: You right. Should I delete it ? I think we don't need it

**Comment by lpriima** [comment]
> I think this looks right. Wasn't `dd.internal.entity_id` using the same prefixed logic? Or was that moved out I guess?

dd.internal.entity_id was here before and i'm not sure how it used, so I didn't change it

File: src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java (hunk 1 above)
Comment (lpriima) [reply src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java]: removed

**Pull request merged**